tests/epyccel/test_epyccel_mod.py

```diff
@@ -23,7 +23,7 @@ def modulo_r_r(x, y):
         return x % y
 
     f = epyccel(modulo_r_r, language=language)
-    x = uniform(high=1e6)
+    x = uniform(low=-1e6, high=1e6)
```

Comment:
I think it would be better to have 2 tests here. One with a positive `x` and one with a negative `x`. If we had this test before your fix then the test would fail some of the time, but not all the time which makes it much harder to identify the problem